docs/docs/HowTo/topics/My-topic-requests.md

```diff
@@ -0,0 +1,52 @@
+# My Topic Requests
+
+The **My Topic Requests** page can be accessed through the **Requests
+-\> My Topic Requests** navigation bar. You can view all requests
+created by you here. This page also provides an audit log and a history
+of the topics and configuration requests created by you.
+
+The Requests are displayed in chronological order by 'Date Requested'
+with the most recent request at the top and the oldest request at the
+bottom. Additionally, on this page, you can find any request you and
+your team have made and its current status.
+
+## Filtering the requests
+
+For easy navigation use the dropdown to filter the requests which you
+are viewing by the status of the request.
+
+-   
+
+    all
+
+    :   View all requests that your team has created.
```

Comment:
The leading colons here can go
![image](https://github.com/aiven/klaw-docs/assets/121855584/1454180a-f3bd-45c0-81c7-3e3baf3a722b)